tailwind.config.ts

```
@@ -98,23 +98,45 @@ const config: Config = withTV({
   			}
   		},
   		screens: {
-  			mobile: `${BREAKPOINTS.mobile}px`,
-  			tablet: `${BREAKPOINTS.tablet}px`,
-  			laptop: `${BREAKPOINTS.laptop}px`,
-  			desktop: `${BREAKPOINTS.desktop}px`,
-  			wide: `${BREAKPOINTS.wide}px`
+  			mobile: '`${BREAKPOINTS.mobile}px`',
```

ESLint

[error] Replace `↹↹↹mobile:·'`${BREAKPOINTS.mobile}px`'` with `······mobile:·"`${BREAKPOINTS.mobile}px`"`

(prettier/prettier)